packages/blade/src/components/Carousel/Indicators/getIndicatorButtonStyles.ts

```diff
@@ -16,6 +16,9 @@ const getIndicatorButtonStyles = ({
     blue: theme.colors.brand.primary[500],
   };
 
+  const easing = castWebType(theme.motion.easing.standard.effective);
```

Comment:
It should be fine, castNativeType doesn't break anything on runtime just casts the type. Though i'll put the easing/duration which are only gonna be run on web inside the `!isReactNative()` check.

Reply:
No it won't because when doing RN typechecking the castWebType will return never (this is expected behaviour). 

<img width="625" alt="image" src="https://github.com/razorpay/blade/assets/35374649/d2d382a0-f39f-4a82-8fb6-7fdc020961f3">